.nuspec/Xamarin.Forms.Platform.WPF.nuspec

```diff
@@ -24,7 +27,10 @@
   <files>
     <!--Icon-->
     <file src="..\Assets\xamarin_128x128.png" target="Assets\" />
-    <file src="..\Xamarin.Forms.Platform.WPF\bin\$Configuration$\Xamarin.Forms.Platform.WPF.dll" target="lib\net45" />
-    <file src="..\Xamarin.Forms.Platform.WPF\bin\$Configuration$\Xamarin.Forms.Platform.WPF.*pdb" target="lib\net45" />
+    <file src="..\Xamarin.Forms.Platform.WPF\bin\$Configuration$\net461\Xamarin.Forms.Platform.WPF.dll" target="lib\net461" />
+    <file src="..\Xamarin.Forms.Platform.WPF\bin\$Configuration$\net461\Xamarin.Forms.Platform.WPF.*pdb" target="lib\net461" />
+    
+    <file src="..\Xamarin.Forms.Platform.WPF\bin\$Configuration$\netcoreapp3.1\Xamarin.Forms.Platform.WPF.dll" target="lib\netcoreapp3.1" />
```

Comment:
will it work with netcoreapp3.0 or netcoreapp5.0?

Reply:
This is targeted at .net core 3.1 and .net 4.6.1 for compatability. 5.0 does not support wpf yet I think. I dont really know if 3.1 and 3.0 are compatible but since 3.0 is already end of life and 3.1 is long term supported I dont see why you would want to use 3.0 anyway :)